recipes/pydantic-zarr/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pydantic-zarr" %}
+{% set version = "0.7.0" %}
+{% set python_min = "3.9" %}
```

Comment:
```suggestion
```

Reply:
I initially didn't have this line, and then the Windows build was failing. But maybe I misinterpreted the cause of the issue? Will try to remove it again.

Reply:
Indeed, failing again now. It seems that setting `python_min` is necessary here?

Reply:
This is usually hiding another problem b/c `python_min` is definitely defined. Can you submit these recipes separately so we can see what is going on here?

Reply:
I submitted `pydantic-zarr` separately in https://github.com/conda-forge/staged-recipes/pull/28800, and the build seems to have worked there, without the `set python_min` as sugggested 🎉.

I'll wait for this other PR to be merged and then remove the `pydantic-zarr` recipe from here.

Reply:
Ok, I removed `pydantic-zarr` and renamed this PR accordingly.